FixedArray.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  JTFixedArray.swift
+//  
+//  A fixed length array type in Swift.
+//
+//  Created by Ben on 26/05/2016.
+
+import Foundation
+
+struct JTFixedArray<T: Comparable> {
+	private let maxSize: Int
+	private let defaultValue: T!
+	private var array: [T]
+	private (set) var count = 0
```

Comment:
In the init method, you give every spot in the array a default value. 
I'm don't really understand what `count` meant to represent?